watershed_extent_update.Rmd

```diff
@@ -16,111 +16,130 @@ The purpose of this document is to establish an agreed upon spatial extent for e
-                 weight = ~rescale(streamorde, to = c(1, 4)),
-                 popup = ~label) %>%
-    addPolylines(data = p3_flowlines_sf, group = "Streams (small)",
-                 color = scico(9, palette = 'davos')[5], opacity = 0.8,
-                 weight = ~rescale(streamorde, to = c(0.25, 1)),
-                 popup = ~label) %>%
-    groupOptions("Streams (small)", zoomLevels = 1:7) %>%
-    groupOptions("Streams (large)", zoomLevels = 8:20) %>%
-    
-    # Add dynamic size gage sites
-    addCircleMarkers(data = p3_gage_sites_sf, group = c("Gage sites (large)"),
-                     color = ~pal_gage(in_HUC8), radius = 5, weight = 2,
-                     popup = ~label) %>%
-    addCircleMarkers(data = p3_gage_sites_sf, group = c("Gage sites (small)"),
-                     color = ~pal_gage(in_HUC8), radius = 3, weight = 1,
-                     popup = ~label) %>%
-    groupOptions("Gage sites (small)", zoomLevels = 1:7) %>%
-    groupOptions("Gage sites (large)", zoomLevels = 8:20) %>%
-    
-    # # Add legend
-    # addLegend(data = p3_gage_sites_sf, group = c("Gage sites"),
-    #           title = "Within saline lake subbasin",
-    #           pal = pal_gage, values = ~in_HUC8,
-    #           position = "bottomright") %>%
-    
-    # Add layer controls
-    addLayersControl(baseGroups = c("Labels", "No Labels"),
-                     overlayGroups = c("Saline lakes", "Lake labels", "Watershed Extent", "Streams (small)", "Streams (large)", "Gage sites (small)", "Gage sites (large)"),
+                        labelOptions = labelOptions(noHide = T, textOnly = T)) %>% 
+    # addLegend(data = , group = c("Watershed"),
+    #            title = "Within saline lake subbasin",
+    #            pal = pal_gage, values = ~in_HUC8,
+    #            position = "bottomright") %>%
+  addLayersControl(baseGroups = c("Labels", "No Labels"),
+                     overlayGroups = c("Saline lakes", "Great Basin HUC6", "Great Basin HUC8", "Great Basin HUC10", "Watershed Extent","Watershed Extent v2",
+                                       "Watershed Extent by huc10", "Lake labels"),
                      position = "topright",
-                     options = layersControlOptions(collapsed = F)) #%>%
-  #hideGroup(c("Streams", "Gage sites"))
-  
-
-  return(leaflet_map)
-  
-}
+                     options = layersControlOptions(collapsed = F))
 
 ```
 
-```{r adapted_function_leaflet_output, echo = FALSE, message =FALSE, warnings = FALSE, out.width = '100%'}
 
-build_map_leaflet_updated_082322(p3_watershed_sf = p2_lake_watersheds_dissolved, 
-                      p3_saline_lakes_sf = p3_saline_lakes_sf, 
-                      p3_flowlines_sf = p3_flowlines_sf, 
-                      p3_gage_sites_sf = p3_gage_sites_sf)
+```{r message =FALSE, echo = FALSE, warnings = FALSE}
 
-## note - a warning appears because Carson sink has not point coords (lat lon) - ignore and can incorporate when loading in carson sink
+leaflet_map_huc
 
```

Comment:
Then add a section that's like:
Next Steps:
- Additional review of watershed boundaries by SMEs
- Generate shapefiles and metadata
- Share with group via Saline Lakes sharepoint (sciencebase?)
- Continued work on site and data inventory